apps/system/test/unit/screen_auto_brightness_test.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+/* global MocksHelper, MockScreenManager, ScreenAutoBrightness */
+
+requireApp('system/test/unit/mock_screen_manager.js');
+require('/js/screen_auto_brightness.js');
+
+var mocksForAutoBrightness = new MocksHelper([
+  'ScreenManager'
```

Comment:
If you are using the `onbrightnesschange` pattern you would not need the mock :)